components/merkledb/src/views/tests.rs

```diff
@@ -681,6 +679,53 @@ fn mutable_and_immutable_prefixed_borrows() {
     _mutable_and_immutable_borrows(&TemporaryDB::new(), PREFIXED_IDX);
 }
 
+#[test]
+fn views_based_on_rc_fork() {
```

Comment:
What are we checking in this test? Does it make sense to split it in a smaller pieces?